SustainabilityOnMars/GuidedNotebooks/pets-in-space/notebook-basics.ipynb

```diff
@@ -0,0 +1,124 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hackathon Challenges and Resources\n",
+    "\n",
+    "These are notebooks and other resources to use as challenges for Callysto hackathons or related events. You're also welcome to use them for other non-commercial purposes.\n",
+    "\n",
+    "Notebooks in this branch were tested on [Callysto Hub](https://hub.callysto.ca).\n",
+    "\n",
+    "### Callysto Hub basics\n",
+    "  \n",
+    "Callysto Hub is a free Jupyter notebook environment provided by [Cybera](https://www.cybera.ca/) and [PIMS](https://www.pims.math.ca/), and it's available [here](https://hub.callysto.ca). \n",
+    "\n",
+    "In order to log in to Callysto  - you can  use your Google account, or a Microsoft account.  \n",
+    "     \n",
+    "Once you have logged in first time - your personal environment will be created, where all your work will be stored.\n",
+    "\n",
+    "\n",
+    "### Copy notebooks to Callysto Hub\n",
```

Comment:
I would delete the "Copy notebooks to Callysto Hub" section as it's probably too advanced for how the students will interact with the notebooks. And the bitly link won't work properly.